docs/shell-tips.md

```diff
@@ -158,16 +191,146 @@ and hit <kbd>ENTER</kbd>/<kbd>RETURN</kbd> at the end.
 
 If you think about it, what is happening here is actually another case of
 character escaping. The newline character (the one that appears when you hit
-<kbd>ENTER</kbd>) usually means "read this command". However, here we want to literally
-have the newline character, and thus the `\<newline>`.
+<kbd>ENTER</kbd>) usually means "read this command". However, here we want to
+literally have the newline character, and thus the `\<newline>`.
```

Comment:
I think it's right. Take a look at [this](http://tldp.org/LDP/abs/html/escapingsection.html).

It's another example of escaping, as you can see.